scripts/monitoring/ops-runner.sh

```diff
@@ -74,21 +92,31 @@ if [ ! -z "$SLEEP" ] ; then
    sleep $(( $RANDOM % $SLEEP  + 1 ))s
 fi
 
+TEMP_FILE=$(mktemp ops-runner-${NAME}-XXXXXXXXXX)
```

Comment:
I would move this to above the cleanup function so that it's always in scope (in case an exit comes before this is defined, the exit won't error).